cgi-bin/ljtestlib.pl

```diff
@@ -0,0 +1,83 @@
+package LJ;
+
+use strict;
```

Comment:
Above this please throw a,

`use v5.10;`

This won't have much practical impact on this particular file, but it clearly communicates to the reader what Perl semantics to expect in this file, in essence. We use this in all new files we're creating for the past few years.

Reply:
done